doc/api/crypto.md

```diff
@@ -2123,6 +2211,61 @@ Computes the Diffie-Hellman secret based on a `privateKey` and a `publicKey`.
 Both keys must have the same `asymmetricKeyType`, which must be one of `'dh'`
 (for Diffie-Hellman), `'ec'` (for ECDH), `'x448'`, or `'x25519'` (for ECDH-ES).
 
+### `crypto.generateKey(type, options, callback)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `type`: {string} The intended use of the generated secret key. Currently
+  accepted values are `'hmac'` and `'aes'`.
+* `options`: {Object}
+  * `length`: {number} The bit length of the key to generate. This must be a
+    value greater than 0.
+    * If `type` is `'hmac'`, the minimum is 1, and the maximum length is
+      2<sup>31</sup>-1. If the value is not a multiple of 8, the generated
+      key will be truncated to `Math.floor(length / 8)`.
+    * If `type` is `'aes'`, the length must be one of `128` or `256`.
```

Comment:
192 is not supported by chromium, but i guess should be supported by node

Reply:
I've gone back and forth on this. The reason it is not supported by Chromium is because 192 is not supported by boringssl, which is also used by electron. We can definitely support 192 but we wouldn't be able to everywhere, and since it's not supported in chromium, there's not likely to be much cryptoapi code out there using it. We can always go back and add it

Reply:
I think it should be in place for node  Boringssl doesn't support aeskw either and we will ship that.

As a consumer of these crypto runtime i know how to deal with degrading support for algorithms, it should not be taken away. Even more so if i can achieve the same with regular crypto

Reply:
Ok, definitely not a problem. Will add it in.